File: 0014-longest-common-prefix/0014-longest-common-prefix.py
class Solution(object):
    def longestCommonPrefix(self, strs):
        """
        :type strs: List[str]
        :rtype: str
        """
        if not strs:
            return ""
        # Take the shortest word in the list
        shortest = min(strs, key=len)
        for i in range(len(shortest)):
            for word in strs:
                if word[i] != shortest[i]:
                    return shortest[:i]
        return shortest
                

File: 0014-longest-common-prefix/0014-longest-common-prefix.java
class Solution {
    public String longestCommonPrefix(String[] strs) {
        Arrays.sort(strs);
        String a="";
        int b;
         if (strs[0].length()>strs[strs.length-1].length()){
            b=strs[strs.length-1].length();
         }
         else{
            b=strs[0].length();
         }
         for (int i=0;i<b;i++){
            if (strs[0].charAt(i)==strs[strs.length-1].charAt(i)){
                a+=strs[0].charAt(i);
            }
            else{
                break;
            }
         } 
         return a;
        }
    }